airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -98,14 +98,16 @@ def format_java_container(dagger_client: dagger.Client, java_code: dagger.Direct
     )
 
 
-def format_js_container(dagger_client: dagger.Client, js_code: dagger.Directory, prettier_version: str = "3.0.3") -> dagger.Container:
+def format_js_container(dagger_client: dagger.Client, js_code: dagger.Directory, prettier_version: str = "3.2.5") -> dagger.Container:
     """Create a Node container with prettier installed with mounted code to format and a cache volume."""
+    warmup_dir = dagger_client.host().directory(".", include=WARM_UP_INCLUSIONS[Formatter.JS], exclude=DEFAULT_FORMAT_IGNORE_LIST)
```

Comment:
I am okay removing these changes — this seems to have no effect, but I don't see how this hurts either. Fine either way.